--- trunk2/configure.ac	2014-11-25 21:32:57.548548603 +0100
+++ trunk/configure.ac	2014-11-25 15:44:47.076156127 +0100
@@ -507,6 +507,8 @@
 	src/plugins/contrib/wxSmithDemo/Makefile
 	src/plugins/contrib/wxSmithIExplore/Makefile
 	src/plugins/contrib/wxSmithPlot/Makefile
+        src/plugins/contrib/wxSmithPlot/wxsmithplot.pc
+        src/plugins/contrib/wxSmithPlot/wxmathplot/Makefile
 	src/plugins/contrib/wxSmithSTC/Makefile
 	src/plugins/contrib/lib_finder/Makefile
 	src/plugins/contrib/lib_finder/lib_finder/Makefile
